test/e2e/resources/xgboost_trainingjob_debugger.yaml

```diff
@@ -68,6 +68,14 @@ spec:
         collection_names: metrics
         num_steps: "10"
         rule_to_invoke: LossNotDecreasing
+  profilerConfig:
+    s3OutputPath: s3://$SAGEMAKER_DATA_BUCKET/sagemaker/training/profiler/
+    profilingIntervalInMilliseconds: 500
+  profilerRuleConfigurations:
+    - ruleConfigurationName: ProfilerReport
+      ruleEvaluatorImage: $DEBUGGER_IMAGE_URI
```

Comment:
should we rename these images or does SageMaker also use "debugger" images for profiler ?

Reply:
Sagemaker also uses debugger images for profiler. Profiler is part of same smdebug library